ChangeSets dupsort

AskAlexSharov opened the pull request:
1 known problem: call_traces stage return "Nonce too high" error on ./cmd/integration,(even if i totally remove cache warmup by changesets...).

AlexeyAkhunov commented on common/changeset/account_changeset_utils.go:
```diff
@@ -2,193 +2,35 @@ package changeset
+func encodeAccounts2(blockN uint64, s *ChangeSet, f func(k, v []byte) error) error {
+	sort.Sort(s)
+	for _, cs := range s.Changes {
+		newK := dbutils.EncodeBlockNumber(blockN)
```
You can take this out of the loop

AlexeyAkhunov commented on common/dbutils/bucket.go:
```diff
@@ -249,6 +249,9 @@ var DeprecatedBuckets = []string{
 	CurrentStateBucketOld1,
 	PlainStateBucketOld1,
 	IntermediateTrieHashBucketOld1,
+	"uncles",
```
Is this a left-over from another change?

AlexeyAkhunov commented:
Looks good to me, I will complete the test of `checkChangeSets` and then a resync of starting from stage5 to see the performance implications

AlexeyAkhunov commented:
I think CI is going crazy building docker on this branch though. Sending build context of 500 Gb ??

mandrigin commented:
yeah, and that is after I updated dockerignore :-/

AskAlexSharov commented:
- Fixed docker problem. 500GB - because some tests are creating database file inside project and I a bit broken in-memory mode.  
- Added to checkChangeSet validation - it now stops if stages Exec or History are lower than block you trying to check
- Continue on checkChangeSet
